mods/wlssb/scripts.js

```diff
@@ -35,4 +35,24 @@ exports.BattleScripts = {
 		}
 		return true;
 	},
+	//Polkadot Bow Memes (Code stolen from Spandan and the Benjamin Butterfree Metagame <3)
+	pokemon: {
+		damage(d, source, effect) {
+			if (!this.hp) return 0;
+			if (d < 1 && d > 0) d = 1;
+			d = Math.floor(d);
+			if (isNaN(d)) return 0;
+			if (d <= 0) return 0;
+			this.hp -= d;
+			if (this.hp <= 0) {
+				d += this.hp;
+				if (this.item === 'polkadotbow' && this.species === 'Lycanroc') {
```

Comment:
Ok @Lycanium-Z what does your mon do anyways?

Reply:
feelsnv

Reply:
... no, hell no.

I asked you simply to make a ballanced game, did you really ahve to do this? we have a deadline and more to do, no time to waste! nerf it properly, or i will myself

Reply:
Alright Alright will do!